### PR TITLE
Ft blog

### PR DESCRIPTION
**Pull request by andela-tpeters**

### CODE REVIEW

File: src/js/components/blog/Blog.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          </div>
+
+          <div className="load-more-container">
+            <i className="fa fa-refresh"></i>
+            <p>Going to classrooms to fetch more articles ...</p>
+          </div>
+        </div>
+      </div>
+    );
+  }
```
Comment (andela-iamadi) [inline src/js/components/blog/Blog.js]: @andela-tpeters  Can we have these broken into smaller components? `BlogHeader` and `BlogLists` could be separate components.

File: src/js/components/common/Header.js [hunk 2]
```diff
@@ -76,6 +76,7 @@ class Header extends React.Component {
               {flagChecks.volunteerPageIsEnabled() && <NavLink path="/volunteer" name="Volunteers" />}
               {flagChecks.curriculumIsEnabled() && <NavLink path="/curriculum" name="Curriculum" />}
               {flagChecks.contactUsIsEnabled() && <NavLink path="/contact-us" name="Contact Us" />}
+              {flagChecks.contactUsIsEnabled() && <NavLink path="/blog" name="Blog" />}
```
Comment (andela-iamadi) [inline src/js/components/common/Header.js]: Great job for hiding this feature behind a flag!! ❤️

There are other parts to flag creation though. Check out [this guide](https://github.com/teencode-org/teencode/blob/ft-blog/tools/README.md) to adding feature flags on the Teencode frontend

File: src/js/routes.js [hunk 3]
```diff
@@ -35,6 +36,7 @@ export default (
              component={ContactUsPage}
              onEnter={redirectIfFlagIsDisabled.bind(null, flagChecks.contactUsIsEnabled())} />
       <Route path="/not-found" component={NotFound} />
+      <Route path="/blog" component={Blog} />
```
Comment (andela-iamadi) [inline src/js/routes.js]: The "/not-found" component should be the last component declared on the root '/' tree. Could we move the `/blog` upwards a bit?

Comment (andela-tpeters) [inline src/js/routes.js]: Resolved

File: src/js/components/common/Header.js (hunk 2 above)
Comment (andela-tpeters) [inline src/js/components/common/Header.js]: Resolved

File: src/js/components/blog/Blog.js (hunk 1 above)
Comment (andela-tpeters) [inline src/js/components/blog/Blog.js]: Resolved

File: src/js/components/blog/BlogList.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+          author="by Rowland Henshaw"
+          summary="The alarms that wake us up, the tools with which we plan our day, our interactions with friends and colleagues, 
+          automations in business processes, you name it"
+        />
+      </div>
+
+      <div className="load-more-container">
+        <i className="fa fa-refresh" />
+        <p>Going to classrooms to fetch more articles ...</p>
+      </div>
```
Comment (andela-iamadi) [inline src/js/components/blog/BlogList.js]: Amazing! One last .. this load more container feels like a separate component too.

And there are no tests yet?

**Pull request merged**